apps/meteor/tests/e2e/page-objects/omnichannel-departments.ts

```diff
@@ -68,6 +68,10 @@ export class OmnichannelDepartments {
 		return this.page.locator('table tr:first-child td:first-child');
 	}
 
+	get btnDeleteFirstRowInTable() {
+		return this.page.locator('table tr:first-child td:nth-child(6) button');
```

Comment:
```suggestion
		return this.page.locator('button[title="Remove"]');
```
does it work?

Reply:
Won't this conflict in the bad case there's other "remove" button around? 👀

Reply:
This line was added but this isn't needed because of the new changes hehe, so I will delete this function